app/Main.hs

```diff
@@ -536,6 +541,14 @@ addFromBower name = do
   where
     stripBowerNamePrefix s = fromMaybe s $ T.stripPrefix "purescript-" s
     mkPackageName' = Bifunctor.first show . mkPackageName . stripBowerNamePrefix
+    unpackSpecifier = second (fmap T.tail . textToMaybe) . T.breakOn "#"
```

Comment:
probably should use Bifunctor.second here, not the import from Arrow

Reply:
Out of curiosity, why is Bifunctor preferred over Arrow?